server/test/api/debtors.js

```diff
@@ -53,4 +53,46 @@ describe('The /debtors API', function () {
       })
       .catch(helpers.handler);
   });
+
+ it.skip('METHOD : GET, PATH : /debtors/:uuid/invoices?balanced=3 returns an error bad value sent', function () {
+    return agent.get('/debtors/3be232f9-a4b9-4af6-984c-5d3f87d5c107/invoices?balanced=3')
+      .then(function (result) {
+        expect(result).to.have.status(400);
+        expect(result.body).to.not.be.empty;
+        expect(result.body).to.have.all.keys('code', 'httpStatus', 'reason');
+      })
+      .catch(helpers.handler);
+  });
+  
+  it('METHOD : GET, PATH : /debtors/:uuid/invoices?balanced=0 returns a list of unbalance invoices of a given debtor', function () {
+    return agent.get('/debtors/3be232f9-a4b9-4af6-984c-5d3f87d5c107/invoices?balanced=0')
```

Comment:
Instead of hard-coding the UUID into this string, it would be more readable and maintainable to put the UUID in a variable and template it in later.  Something like this:
```js
// at the top of the describe() block
var debtorUuid = 'be232f9-a4b9-4af6-984c-5d3f87d5c107';

// ... later in the test script ...
it ('some description', function () {
  var target = '/debtors/:uuid/invoices?balanced=0'.replace(':uuid', debtorUuid);
  return agent.get(target)
  .then(function (res) { /* test conditions, etc.. */ });
});
```
Alternative, if you don't like using `replace()` you can just concatenate `'/debtors/' + debtorUuid + '/invoices?balanced=0'`.